README.md

```diff
@@ -11,40 +11,19 @@ Service Manual Publisher is used for publishing and maintaining https://www.gov.
 
 ## Technical documentation
 
-PostgreSQL-backed Rails publishing application for internal use, with no public facing aspect.
+This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
 
 There is some [disparity between the content as it appears in the database, and the content as it appears in the publishing api](doc/arch/002-disparity-between-database-and-publishing-api.md). This will need to be addressed if we switch to using the Publishing API as our main data store in the future.
```

Comment:
We could move these to a later "Further documentation" section, and maybe help out @ChrisBAshton by renaming `doc/ -> docs`.

Reply:
Good point, solved in 78bbf1a and d84669f.